components/widgets/inputs/input/input.tsx

```diff
@@ -63,7 +68,13 @@ const Input = React.forwardRef((
     const {formatMessage} = useIntl();
 
     const [focused, setFocused] = useState(false);
-    const [stateError, setStateError] = useState('');
+    const [customInputLabel, setCustomInputLabel] = useState<CustomMessageInputType>(null);
+
+    useEffect(() => {
+        if (customMessage !== undefined && customMessage !== null && customMessage.value !== '') {
```

Comment:
Maybe `customMessage?.value !== ''` could be enough.

Reply:
customMessage is conditional prop and per se could be undefined, and it gives also the option to be null for customInputLabel, so not a problem to have this guards.